corehq/apps/app_manager/dbaccessors.py

```diff
@@ -190,7 +190,7 @@ def get_app_cached(domain, app_id):
     if not app:
         app = get_app(domain, app_id)
         if app.copy_of:
-            cache.set(key, app, timeout)
+            cache.set(key, app)
```

Comment:
dont you need to use '24 * 3600' as timeout?

Reply:
This is tiered_cache, the timeout is already set in lines above 188 per each cache